### DISCUSSION TITLE
How can I Limit the Number of Country Displayed I want to Have only one or Two Countries In the  search filed

### DISCUSSION THREAD

**Discussion by Smile-Room** [natintosh/intl-phone-number-input#325] [Q&A]
first I Love this package, I would like to Thank the Creators, But I have one question. How can I Limit the Number  of Country Displayed
I want to Have only one or Two Countries I search filed. how can I achieve this, Thanks In advance

**Comment by EthraaAlmaerifa** [comment]
you can use the `countries` parameter
```
InternationalPhoneNumberInput(
     countries: ['SA', 'NG'],
)
```
